app/cells/move_information/show.slim

```diff
@@ -21,20 +21,22 @@
       = f.radio_toggle :has_destinations do
         #establishments
         #error_move_destinations
-        - f.fields_for :destinations do |df|
-          .form-group.optional-section.panel.panel-border-narrow
-            = df.hidden_field :id
-            = df.text_field :establishment
-            .inline
-              = df.label_with_radio(:must_return, 'Must return', 'must_return')
-              = df.label_with_radio(:must_return, 'Must NOT return', 'must_not_return')
-              = df.label_with_radio(:must_return, 'Clear selection', 'unknown')
-            = df.text_area :reasons
-            .form-group
-              = df.label :_delete, class: 'styled-as-link'
-              = df.check_box :_delete
+          .form-group
+            - f.fields_for :destinations do |df|
+              .form-group.optional-section.panel.panel-border-narrow
+                = df.hidden_field :id
+                .form-group
+                  .establishment-field
+                    = df.text_field :establishment
+                  .must-return-field
+                    = df.label_with_radio(:must_return, 'Must return', 'must_return')
+                    = df.label_with_radio(:must_return, 'Must NOT return', 'must_not_return')
```

Comment:
We have a ticket in this sprint for introducing all the copy in the translation file